modules/dr-restoring-cluster-state.adoc

```diff
@@ -318,7 +318,7 @@ Note that it might take several minutes after completing this procedure for all
 [id="dr-scenario-cluster-state-issues_{context}"]
 = Issues and workarounds for restoring a persistent storage state
 
-If your {product-title} cluster uses persistent storage of any form, a state of the world is typically stored outside etcd. It might be an Elasticsearch cluster running in a pod or a database running in a `StatefulSet` object. When you restore from an etcd backup, the status of the workloads in {product-title} is also restored. However, if the etcd snapshot is old, the status might be invalid or outdated.
+If your {product-title} cluster uses persistent storage of any form, the state of the cluster is typically stored outside etcd. It might be an Elasticsearch cluster running in a pod or a database running in a `StatefulSet` object. When you restore from an etcd backup, the status of the workloads in {product-title} is also restored. However, if the etcd snapshot is old, the status might be invalid or outdated.
```

Comment:
hmm, I preferred "a state of the cluster", because now cluster is stored in two places. One in etcd and another in volumes that are outside etcd.

Reply:
@gnufied's comment makes sense to me